build.sbt

```diff
@@ -21,20 +21,20 @@ val compilerOptions = Seq(
 ThisBuild / scalacOptions ++= compilerOptions
 
 val playJsonVersion = "2.9.4"
-val specsVersion: String = "4.8.3"
+val specsVersion: String = "4.20.0"
 val awsSdkVersion: String = "1.12.482"
 val doobieVersion: String = "0.13.4"
 val catsVersion: String = "2.9.0"
 val okHttpVersion: String = "4.11.0"
 val paClientVersion: String = "7.0.7"
 val apacheThrift: String = "0.15.0"
-val jacksonDatabind: String = "2.15.1"
-val jacksonCbor: String = "2.15.1"
-val jacksonScalaModule: String = "2.15.1"
-val simpleConfigurationVersion: String = "1.5.6"
+val jacksonDatabind: String = "2.15.2"
+val jacksonCbor: String = "2.15.2"
+val jacksonScalaModule: String = "2.15.2"
+val simpleConfigurationVersion: String = "1.5.7"
```

Comment:
@rtyley - Hi there, another scala-steward question. In this repository we have defined a [.scala-steward.conf](https://github.com/guardian/mobile-n10n/blob/main/.scala-steward.conf) which should be preventing scala-steward from bumping some dependencies. We use the scala-steward.conf file in cases where we know the library is incompatible and we're dealing with these larger/more complicated upgrades as health tasks, whereas we tend to deal with scala-steward PRs during our weekly dependency update mobbing.

In this PR we can see that the config defined in our scala-steward.conf isn't being respected. We have equivalent [config](https://github.com/guardian/mobile-apps-api/blob/main/.scala-steward.conf) in MAPI, which does appear to be respected.

Do you have any clues as to why this is happening?

Reply:
It turns out that Scala Steward is giving us the clue, in the description for this PR...!

![image](https://github.com/guardian/mobile-n10n/assets/52038/e9cf7663-e6be-45ee-8444-b354f647171d)

The warning is hidden unobtrusively near the bottom - there was recently [an update to Scala Steward](https://github.com/scala-steward-org/scala-steward/pull/2986) to make the message slightly more visible - with an addtional `❗`  - which you can see in [a more recent PR](https://github.com/guardian/mobile-n10n/pull/1132) on this repo:

![image](https://github.com/guardian/mobile-n10n/assets/52038/68f1f860-220a-4594-97b2-0d2ce55409af)

The message says:

```
❗ Note that the Scala Steward config file `.scala-steward.conf` wasn't parsed correctly
DecodingFailure at .updates.pin.groupId: Missing required field
```

It doesn't mention a line number, but if you look closely at the file, you can see that one entry (added with 7683204b63ea4873d81d672cee77e3e2a2674f53 in #1122 2 weeks ago) _doesn't_ have a `groupId` - it actually says 'group**d**Id'

https://github.com/guardian/mobile-n10n/blob/bbf082ca5afd8721c4e058d4fb1b395fd6f3ca58/.scala-steward.conf#L20

That single typo meant that the entire config file was rejected - personally, I think Scala Steward should have refused to operate on the repo at that point, but it's current behaviour is to continue on - and so if you fix the type, your config will be respected once again...!